pkg/scheduler/scheduler_test.go

```diff
@@ -205,7 +199,7 @@ func TestSchedulerCreation(t *testing.T) {
 	}
 }
 
-func TestScheduler(t *testing.T) {
+func TestSchedulerSqueduleOne(t *testing.T) {
```

Comment:
A typo here. We can just name it TestScheduleOne.

Reply:
uhm... it was intentional. I would say both work, because we are testing `Scheduler.scheduleOne`.

Reply:
Fx the typo then :) TestSchedulerSqueduleOne -> TestSchedulerScheduleOne

Reply:
lol, I didn't see it.